documentation/scripts/documentation_config.ts

```diff
@@ -264,6 +264,21 @@ export const Snippets: AutocompleteSnippet[] = [
     content: 'Send data to an API.',
     codeFile: './samples/snippets/fetcher/post.ts',
   },
+  {
+    triggerTokens: ['addSkill', 'Skill'],
+    content: 'Adds a skill to guide the LLM.',
+    codeFile: './samples/snippets/skill/add_skill.ts',
+  },
+  {
+    triggerTokens: ['setSkillEntrypoints', 'SkillEntrypoints'],
+    content: 'Sets the entrypoints for skills in the Pack.',
+    codeFile: './samples/snippets/skill/set_skill_entrypoints.ts',
+  },
+  {
+    triggerTokens: ['addMcpServer', 'addMCPServer', 'McpServer', 'MCP'],
```

Comment:
```suggestion
    triggerTokens: ['addMCPServer', 'McpServer', 'MCP'],
```
That first entry seems duplicative.